### DISCUSSION TITLE
Train Models

### DISCUSSION THREAD

**Discussion by HyakuzaO** [kyleskom/NBA-Machine-Learning-Sports-Betting#415] [Q&A]
Hello guys,we only train the XGboost Model or we must train the NN model as well?

Ty for All the help and answers.

**Comment by HyakuzaO** [comment]
@kyleskom ?

**Comment by HyakuzaO** [comment]
Someone?

**Reply by STRATZ-Ken** [reply]
Its a free product.  Pay someone on Upwork or wait. Get some patience.

**Reply by HyakuzaO** [reply]
I know it's a free product which I appreciate a lot. I just asked a question so someone who understands could answer me. I don't see the need for your comment and for you to waste your time commenting when you didn't answer me, and I only asked a simple question with no ill intent. But I understand that comment. Thank you anyway.

**Reply by STRATZ-Ken** [reply]
People have notifications on in this repo. including myself. I get an email on every comment and update.  I dont need 3 emails within 4 hours.

Also, this is not the first time your doing this.

**Reply by HyakuzaO** [reply]
Like i said. Ty anyway.

**Comment by kyleskom** [comment]
You can train either or both. They are independent.
